projects/packages/connection/src/class-error-handler.php

```diff
@@ -702,11 +702,15 @@ public function generic_admin_notice_error() {
 			return;
 		}
 
-		?>
-		<div class="notice notice-error is-dismissible jetpack-message jp-connect" style="display:block !important;">
-			<p><?php echo esc_html( $message ); ?></p>
-		</div>
-		<?php
+		wp_admin_notice(
```

Comment:
@sergeymitr I wasn't able to trigger this notice (using the Jetpack Debug helper). Do you know how those notices are triggered, by any chance?

Reply:
I guess the easiest way to trigger this using the Jetpack Debug helper would be:

- Enable the Broken Token Utilities
- Go to Connection Errors
<img width="329" alt="Screenshot 2024-04-25 at 12 41 41" src="https://github.com/Automattic/jetpack/assets/1758399/e3941db8-3fae-405b-ad4c-12f52c974ba2">

- Create a fake verified user token error
<img width="512" alt="Screenshot 2024-04-25 at 12 38 52" src="https://github.com/Automattic/jetpack/assets/1758399/260906a3-d8f0-4862-87a7-ad89e09539d8">